Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -19,7 +19,8 @@ final class WebSocket {
     /// A WebSocket events callback.
     var onEvent: Client.OnEvent = { _ in }
     
-    private let webSocket: Starscream.WebSocket
+    let webSocket: Starscream.WebSocket
```

Comment:
It should be private.

Reply:
I need to access `webSocket.callbackQueue` from `Client`?

Reply:
Please limit yourself and you will find a solution. As we discussed in the comments above when you move all work with events into WebSocket (where it should be) you won't need to make it internal.